src/Modules/Workspaces/Commands.Workspaces/help/Set-PowerBIWorkspace.md

```diff
@@ -51,6 +56,21 @@ If the current user is an administrator, this will update the specified workspac
 
 ## PARAMETERS
 
+### -CapacityId
+The capacity object id which a workspace will be assigned to. If CapacityId is set to "00000000-0000-0000-0000-000000000000", the workspace will be migrated to shared capacity.
```

Comment:
replace 00000000-0000-0000-0000-000000000000 with [guid]::Empty

Reply:
Somehow I didn't get your comments and I merged it already. Let me open another PR to address this.